presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/HiveCommitHandle.java

```diff
@@ -47,7 +49,10 @@ public HiveCommitHandle(Map<SchemaTableName, List<DateTime>> lastDataCommitTimes
     @Override
     public String getSerializedCommitOutput(SchemaTableName table)
     {
-        Optional<String> serializedCommitOutput = JSON_CODEC.toJsonWithLengthLimit(lastDataCommitTimes.get(table), JSON_LENGTH_LIMIT);
+        List<Long> commitTimes = lastDataCommitTimes.getOrDefault(table, ImmutableList.of()).stream()
+                .map(commitTime -> commitTime.getMillis() / MILLISECONDS_IN_SECOND)
```

Comment:
`TimeUnit.MILLISECONDS.toSeconds` can be used instead.

Reply:
> `TimeUnit.MILLISECONDS.toSeconds` can be used instead.

Good suggestion; have updated.